src/main/kotlin/no/nav/sifinnsynapi/sak/behandling/BehandlingDAO.kt

```diff
@@ -0,0 +1,27 @@
+package no.nav.sifinnsynapi.sak.behandling
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import org.hibernate.annotations.JdbcTypeCode
+import org.hibernate.type.SqlTypes
+import org.springframework.data.annotation.CreatedDate
+import java.time.ZoneOffset.UTC
+import java.time.ZonedDateTime
+import java.util.UUID
+
+@Entity(name = "behandling")
+data class BehandlingDAO(
+    @Column(name = "behandling_id") @Id val behandlingId: UUID,
+    @Column(name = "søker_aktør_id") val søkerAktørId: String,
+    @Column(name = "pleietrengende_aktør_id") val pleietrengendeAktørId: String,
+    @Column(name = "saksnummer") val saksnummer: String,
+    @Column(name = "ytelsetype_kode") val ytelsetypeKode: String,
```

Comment:
Hvis du ønsker at denne fortsatt skal være type enum, men kodestring i DB, så kan du vurdere å bruke @Convert
```    
@Column(name = "ytelsetype_kode")
@Convert(converter = FagsakYtelseTypeConverter::class) 
val ytelsetypeKode: String
```
Og selve konverteren er ganske enkel, kopiert fra formidling. 
```@Converter
internal abstract class KodeverdiConverter<T : Kodeverdi> : AttributeConverter<T, String> {
    override fun convertToDatabaseColumn(attribute: T?): String? = attribute?.kode
}


internal class FagsakYtelseTypeConverter : KodeverdiConverter<FagsakYtelseType>() {
    override fun convertToEntityAttribute(dbData: String?): FagsakYtelseType? =
            if (dbData == null) null else FagsakYtelseType.fraKode(dbData)
}
```

Reply:
Det er en god ide 👍 
ytelsetypeKode er fortsatt String da?

Reply:
sorry jeg skrev feil det skal være enum type ikke string :D var hele meningen

Reply:
Ingen problem. Da tenker jeg at enum er bedre 👍

Reply:
Jeg ser disse converterne bruker Kodeverdi fra kodeverk. Ble det ikke nevnt at de skulle fjernes om ikke lenge?
Isåfall vil jeg anbefale å ikke introdusere teknisk gjeld. Hva tenker du?

Reply:
jeg tror de skal bare endre annotasjonen og ta i bruk JsonValue. Evt. kan du bare lage konverterter på FagsakYtelseType istedenfor Kodeverdi. Men ja aner ikke hva de ender opp med så kanskje greit å ta selve FagsakYtelseType.

Reply:
Ja, jeg tror jeg bare lager en converter for FagsakYtelseType :)

Reply:
Converter med enum er lagt til i siste commit.